### PR TITLE
[change] Remove customization for checks and replies #336

### LINKED ISSUE #336: [change!] Remove customization for checks and replies
The following customizations for check and replies are no longer maintained by the contributor who originally added them:

- is_active field
- valid_until field
- choices for attribute
- logic which implements password field in the admin
- documentation mentioning these features

I am not aware of anyone else using it and I think are overkill to keep in the core of OpenWISP RADIUS.
Since these customizations can be implemented by extending/forking OpenWISP RADIUS, we should remove these to reduce maintenance overhead.

The choice field on the attribute field also limits the flexibility of the checks/replies.

### PR DESCRIPTION
**Pull request by totallynotvaishnav**
Closes #336

### CODE REVIEW

**Comment by totallynotvaishnav** [comment]
@devkapilbansal could you review it?

File: openwisp_radius/base/forms.py [hunk 1]
```diff
@@ -43,7 +43,6 @@ class RadiusCheckForm(ModeSwitcherForm):
         label=_('Value'),
         required=False,
         max_length=radcheck_value_field.max_length,
-        widget=forms.PasswordInput(),
```
Comment (devkapilbansal) [inline openwisp_radius/base/forms.py:46]: Why this field is converted to CharField

File: openwisp_radius/admin.py [hunk 2]
```diff
@@ -80,19 +76,15 @@ class RadiusCheckAdmin(MultitenantAdminMixin, TimeStampedEditableAdmin):
...
-        'is_active',
-        'valid_until',
         'created',
         'modified',
     ]
     search_fields = ['username', 'value']
     list_filter = [
         DuplicateListFilter,
         ('organization', MultitenantOrgFilter),
-        ExpiredListFilter,
```
Comment (devkapilbansal) [inline openwisp_radius/admin.py:92]: This filter should also be removed from all filters

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
Thanks @totallynotvaishnav 
I haven't tested it but looks like the PR is in right direction.

Here, are some changes that I can suggest :point_down: 

P.S. :- Documentation needs to be updated too

File: openwisp_radius/base/forms.py (hunk 1 above)
Comment (totallynotvaishnav) [reply openwisp_radius/base/forms.py:46]: My bad. I thought it was no longer needed.

File: openwisp_radius/admin.py (hunk 2 above)
Comment (totallynotvaishnav) [reply openwisp_radius/admin.py:92]: Will remove it.

File: openwisp_radius/admin.py [hunk 3]
```diff
@@ -19,12 +19,7 @@
 )
 
 from . import settings as app_settings
-from .base.admin_actions import disable_action, enable_action
```
Comment (devkapilbansal) [inline openwisp_radius/admin.py:22]: I can't see these two removed anywhere

Comment (totallynotvaishnav) [reply openwisp_radius/admin.py:22]: I removed it in line 115.

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@totallynotvaishnav I would suggest you to run tests locally too. Also, coverage can be a go to way if you want to know whether the code written is covered by tests or not.

For e.g. :- See that the [coverage for `admin actions` file is decreased to 0](https://coveralls.io/jobs/93058523/source_files/16359972336)

**Comment by totallynotvaishnav** [comment]
> @totallynotvaishnav I would suggest you to run tests locally too. Also, coverage can be a go to way if you want to know whether the code written is covered by tests or not.
>

**Comment by totallynotvaishnav** [comment]
> I would suggest you to run tests locally too
All tests are successful in local.

**Review by devkapilbansal — COMMENTED** [review]
@totallynotvaishnav tests pass successfully now. I rebased your branch and made minor improvements. Will check your changes soon

P.S. :- I saw some noisy output in tests [here](https://github.com/openwisp/openwisp-radius/runs/5066168269?check_suite_focus=true#step:9:187). Can you please open an issue for this ? Would be great if a PR is opened too.

**Comment by totallynotvaishnav** [comment]
> Can you please open an issue for this ? Would be great if a PR is opened too.

Sure, will do.

**Comment by totallynotvaishnav** [comment]
> I can find some instances where `is_active` field is present. It is there in docs too

@devkapilbansal I don't see any `is_active` field related to checks. Could you link to any reference?

**Review by nemesisdesign — COMMENTED** [review]
It looks good, thank you @totallynotvaishnav @devkapilbansal :+1:!
I have tested it a bit and will test more in the next days.

**Comment by nemesisdesign** [comment]
There were quite a few things remaining to be deleted, I took are of those in https://github.com/openwisp/openwisp-radius/commit/56cf9f15a56808fa593e38a33f60bc2221d4f768.

**Pull request merged**